assets/src/scripts/components/map/nldiMapping.js

```diff
@@ -1,7 +1,12 @@
 import { geoJson, circleMarker } from 'leaflet';
-import { get } from '../../ajax';
-import config from '../../config';
-import { select } from 'd3-selection';
+import {
```

Comment:
Same as above. Let's try to get all on a single line.

Reply:
Will do.